### ISSUE TITLE
Unexplainable loss WGAN

### ISSUE THREAD

**Issue by lennert130994** [eriklindernoren/Keras-GAN#172]
I am training the WGAN on 3D point cloud data of human heads, but the loss function of the critic and the generator suddenly stabalize. Can anyone explain why this is happening.
Situation is shown in the image below

![image](https://user-images.githubusercontent.com/23079356/61936694-8948a600-af8d-11e9-8643-037f93153ea0.png)
